reference/checkouts.v3.yml

```diff
@@ -1009,18 +981,15 @@ definitions:
                       quantity:
```

Comment:
` physical_items` element is missing the following properties: `is_mutable`, `parent_id`.

Reply:
yes, you are correct, added them